app/views/charge_types/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<% if can?(:read, @charge_type) %>
+  <%- model_class = ChargeType -%>
+  <div class="page-header">
+    <h1><%=t '.title', :default => model_class.model_name.human %></h1>
+  </div>
+
+  <dl class="dl-horizontal">
+    <dt><strong><%= model_class.human_attribute_name(:name) %>:</strong></dt>
+    <dd><%= @task.name %></dd>
```

Comment:
Actually the show page should be removed for now since there's no way to get to it and it is redundant.